source/manual/alerts/email-alerts-travel-medical.html.md

```diff
@@ -6,79 +6,90 @@ layout: manual_layout
+* If an email matching the details from the logs appears in
+  [email-alert-api-alert-listener], check [Notify's service
+  status](https://status.notifications.service.gov.uk/).
+
 * Using the Content ID from the logs, get send/delivery statistics from
-  email-alert-api using the [email statistics rake task][] in email-alert-api
-  to determine whether there was a problem sending the emails to Notify, or
-  problems returned from Notify after it attempted to deliver them.
-* determine whether Email Alert API has a backlog of work to do using the
-  [Sidekiq dashboard][] and [Email Alert API Technical
-  dashboard][tech dashboard] - the email may just be delayed;
-* whether the change was a result of a new type of medical safety content
-  sub-type, this was the previous cause of an [incident][checkbox-incident] and
-  is recorded as [GOV.UK Tech Debt][checkbox tech debt];
-* check the [Kibana logs][] and [Sentry][] for any errors or clues;
-* if all else fails you may need to investigate the Email Alert API database
-  to determine whether the content change was received and what state it is in.
-
-## Resending medical safety emails
-
-If you need to force the sending of a travel advice email alert, run the
-`email_alerts:trigger[PUT_EDITION_ID_HERE]` rake task in Travel Advice
-Publisher.
-
-The edition ID of the travel advice content item can be found in the
-URL of the country's edit page in Travel Advice Publisher and looks like
-`fedc13e231ccd7d63e1abf65`.
-
-## Resending travel advice emails
-
-If you need to force the sending of a travel advice email alert, run the
-`email_alerts:trigger[PUT_EDITION_ID_HERE]` rake task in Travel Advice
-Publisher.
-
-The edition ID of the travel advice content item can be found in the
-URL of the country's edit page in Travel Advice Publisher and looks like
-`fedc13e231ccd7d63e1abf65`.
+  email-alert-api using the [email statistics rake task] to determine whether
+  there was a problem sending the emails to Notify, or problems returned from
+  Notify after it attempted to deliver them.
+
+* Determine whether Email Alert API has a backlog of work to do using the
+  [Sidekiq dashboard] and [email-alert-api dashboard]. Is there a backlog? If
+  so, is it diminishing / are the workers making sufficient progress?
+
+* Previously, the introduction of a new type of medical safety content
+  sub-type caused an [outage][checkbox-incident]; see ["all" checkbox is
+  misleading for finder_email_signups](https://trello.com/c/v2ees2fD).
+
+* Check [logs in
+  Kibana](https://kibana.logit.io/s/13d1a0b1-f54f-407b-a4e5-f53ba653fac3/goto/8562a5cef62e374c0a112d104e5b3f4d?security_tenant=global)
+  and events in [Sentry] for errors or clues.
+
+* If all else fails, investigate the Email Alert API database to determine
+  whether the content change was received and what state it is in.
+
+## Resend a medical safety alert
+
+TODO: add necessary details here. What's the app and rake task for resending
```

Comment:
This was previously identical to "Resend a travel advice alert", so I assumed it was a copy-pasto. Presumably it's something to do with specialist-publisher?